src/index.ts

```diff
@@ -137,6 +148,32 @@ export class WaxJS {
     return this.user.account;
   }
 
+  private async getChainInfoByChainName(chainName: string): Promise<any> {
+    const response = await this.rpc.get_table_rows({
```

Comment:
this call should be using a different rpc member that is used to reference the mainnet/chain that holds the sidechain registry contract, so we need to add a `this.registryRpc` member

Reply:
updated base on comment 👍